solr/core/src/java/org/apache/solr/schema/IndexSchema.java

```diff
@@ -107,6 +107,8 @@
   public static final String LUCENE_MATCH_VERSION_PARAM = "luceneMatchVersion";
   public static final String MAX_CHARS = "maxChars";
   public static final String NAME = "name";
+  public static final String PARENT_FIELD_NAME = "_NEST_PARENT_";
```

Comment:
In one of my earliest review comments I suggested renaming both constants to start with `_NEST_` so that they are more clearly grouped together in their purpose.  WDYT?

Reply:
Sounds good to me :+1: